core/src/main/java/org/apache/calcite/util/RelToSqlConverterUtil.java

```diff
@@ -68,6 +68,43 @@ public static void unparseHiveTrim(
     }
   }
 
+  /**
+   * For usage of TRIM, LTRIM and RTRIM in BQ see
+   * <a href="https://cloud.google.com/bigquery/docs/reference/standard-sql/functions-and-operators#trim">
+   * BQ Trim Function</a>,
+   * <a href="https://prestodb.io/docs/current/functions/string.html#trim-string-varchar">
+   * Presto Trim Function</a>.
+   */
+  public static void unparseTrim(SqlWriter writer, SqlCall call, int leftPrec,
```

Comment:
if this function is specific to BigQuery, the function name should reflect that.
I would not abbreviate BQ in the JavaDoc, why force people to try to guess what it is when you can just write it down?

Reply:
Yes，make sense， this method not only used for BigQuery，can be used by any need convert trim(both from) to trim/ltrim/rtrim functions, so method name may not need to be changed, I had updated the bbreviate BQ in JavaDoc.Thanks for suggestion @mihaibudiu

Reply:
the current name is too generic, this function is not suitable for unparsingTrim for all dialects, as the name would suggest
I think unparseTrimBigQuery is better, or unparseTrimLR if you want.
Many dialects can share it.

Reply:
Yes, I thought unparseTrimLR is better, so changed the function to unparseTrimLR~ @mihaibudiu